_config.yml

```diff
@@ -56,7 +56,7 @@ paginate_path: "blog/page:num"
 markdown: kramdown
 permalink: /:categories/:title/
 highlight: rouge
-gems: [jekyll-paginate]
+plugins: [jekyll-paginate]
```

Comment:
@Phlow This has a side effect, users must use Jekyll 3.5+ (by default) when using this theme. Which I guess is a good thing, keeping everyone on top versions.